integration_tests/test_poller_integration.py

```diff
@@ -623,10 +624,286 @@ def test_inline_host_present_in_group(self, request, setup_splunk):
 
 
 def run_retried_single_search(setup_splunk, search_string, retries):
-    for i in range(retries):
+    for _ in range(retries):
         result_count, metric_count = splunk_single_search(setup_splunk, search_string)
         if result_count or metric_count:
             return result_count, metric_count
         logger.info("No results returned from search. Retrying in 2 seconds...")
         time.sleep(2)
     return 0, 0
+
+
+@pytest.fixture(scope="class")
+def setup_single_gt_and_lt_profiles(request):
```

Comment:
Can you put in the descriptions of the tests what is the expected result? Because it is hard to know as the data is hidden in snmprec file. This will help with debugging in the future.